docs/docs/contributing/code/technical_logistics.mdx

```diff
@@ -1,36 +1,9 @@
----
-sidebar_position: 1
----
-# Contribute Code
+# Technical logistics
```

Comment:
technical logistics sounds vague, maybe one of Setup, Local setup, Setting up repo, Get set up, ...

Reply:
Will go with `Quickstart`

Reply:
Actually just `setup`